browser/about_flags.cc

```diff
@@ -704,6 +705,14 @@
           kOsAll,                                                              \
           FEATURE_VALUE_TYPE(brave_sync::features::kBraveSyncSendAllHistory),  \
       },                                                                       \
+      {                                                                        \
+          "brave-copy-clean-link",                                             \
+          "Enable copy clean link feature",                                    \
```

Comment:
This might lead users into thinking that it disables copy-clean-link entirely. I'd suggest:
```
"brave-copy-clean-link-by-default"
"Default to copy clean link"

Reply:
done